src/lib/eslint/TypescriptEslintStrategy.ts

```diff
@@ -67,16 +67,18 @@ export class TypescriptEslintStrategy implements EslintStrategy {
 			.replace('index.js', path.join('configs', 'eslint-recommended.js'));
 		// The extended base config has an `overrides` property which is a singleton array of the rule configs object
 		// we actually want.
-		const rawExtendedConfig = (await import(pathToExtendedBaseConfig)) as {overrides: ESRuleConfig[]};
-		this.extendedEslintConfig = rawExtendedConfig.overrides[0];
+		const rawExtendedConfig = (await import(pathToExtendedBaseConfig)) as {default: {overrides: ESRuleConfig[]}};
+		this.extendedEslintConfig = rawExtendedConfig.default.overrides[0];
```

Comment:
I'm not sure how things worked before with await import. But with the new node module resolution changes... we now need to do things the right way according to the ECMAScript Modules (ESM) system where we must use the default accessor property to get the default export.